daemon/algod/api/algod.oas2.json

```
@@ -3560,6 +3602,25 @@
         }
       }
     },
+    "BoxReference": {
+      "description": "Box descriptor describes a Box.",
```

This is more similar to how the other new Reference types describe themselves.
```suggestion
      "description": "References a box of an application.",
```

---

Accepted in f343aa459